src/Selector/Selector.module.scss

```diff
@@ -0,0 +1,79 @@
+@import '../mixins.scss';
+@import '../variables.scss';
+
+.wrapper {
+    display: flex;
+    position: relative;
+    height: 32px;
+    box-sizing: border-box;
+    border-radius: $border-radius;
+    align-items: center;
+}
+
+.btn {
+    @include text-normal;
+
+    flex: 1;
+    position: relative;
+    border: none;
+    color: $text-grey;
+    height: 100%;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    cursor: pointer;
+    outline: none;
+
+    background-color: transparent;
+    z-index: 2;
+}
+
+.selected {
+    color: $text-green;
+
+    &::before {
+        display: inline-block;
+        content: '';
+        width: 100%;
+        position: absolute;
+        top: -1px;
+        left: -1px;
+        bottom: -1px;
+        background: $btn-background-light-green;
+        // box-sizing: border-box;
+        border-radius: $border-radius;
+        z-index: -1;
+    }
+    &:last-of-type::before{
+        left: auto;
+        right: -2px;
+    }
+}
+
+.disabled {
+    color: #888;
+    cursor: not-allowed;
+}
+
+.border {
+    &.wrapper {
+        border: 1px solid $light-gray3;
+        & .selected::before {
+            border: 1px solid $green;
+        }
+    }
+}
+
+:global(.night-mode) {
+    .wrapper,
+    .selected {
+        color: #a0a0a0;
+        background-color: #252932;
```

Comment:
@DmitriVanGuard In design system we got dark mode colors.

Reply:
@Partysun They are the same as the `day mode`